release-notes/Cloud/dataminer_services_change_log_2025.md

```diff
@@ -9,6 +9,18 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+### 05 May 2025 - 5 May 2025 - Enhancement - Catalog - Deletion of items by publishing organization [ID 42793]
+
+Users who are part of the publishing organization are able to delete a catalog item, providing a reason why.
```

Comment:
```suggestion
If you are a member of the organization that published a specific Catalog item, you can now delete that Catalog item. When you do so, you will have to provide the reason for the deletion.
```

To make this more user-friendly, it might also be better if we went into a bit more detail as to how you can delete the item.